tests/hooks/event-conversion-utils.spec.ts

```diff
@@ -17,7 +17,7 @@ type RecordFields = {
   picture?: object[],
   types?: string[]|string,
   location?: string,
-  region?: string[]|string,
+  regions?: string[]|string,
```

Comment:
Just wondering why not just `string[]`? Does the Airtable API send just a string instead of a string inside an array if only one region is selected?